server/routes/auth.js

```diff
@@ -22,4 +23,6 @@ router.post('/changeavatar', verifyToken, userSettings.handleChangeAvatar);
 
 router.get('/authLogin', authUser);
 
+router.get('/logout', verifyToken, logout.handleLogout);
```

Comment:
we probably don't need to verifyToken for this route - because in the scenario where it happens to be an invalid token, we would still want to log them out :)